src/content/docs/query-your-data/explore-query-data/dashboards/dashboard-template-variables.mdx

```diff
@@ -637,6 +645,48 @@ This results in the values for the last 30 minutes. When you set the time picker
 
 </Collapser>
 
+<Collapser
+    id="from-data-types"
+    title="Use variables to select your `FROM` data type"
+>
+
+  Sometimes you have queries that apply the same rules to different types of data. Here are examples of two widgets that are identical except for the data type:
+
+  
+  * WIDGET 1: 
+      ``` 
```

Comment:
@rhetoric101 
the only suggestion I would make is to use the same language syntax as other examples in this page.
Thanks!

Reply:
I switched these to single-line snippets.